components/channel_header/channel_header.js

```diff
@@ -98,6 +96,9 @@ export default class ChannelHeader extends React.PureComponent {
 
     componentDidMount() {
         this.props.actions.getCustomEmojisInText(this.props.channel ? this.props.channel.header : '');
+        if (this.props.dmUser) {
+            this.props.actions.loadBot(this.props.dmUser.id);
```

Comment:
Is there some way that we can only do this for users that we know are bots and when we haven't loaded the bot already? As it is, this adds an extra API call to every channel change when viewing DMs